src/matter/cluster/TemperatureMeasurementCluster.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Attribute, Cluster, Command, TlvNoArguments, TlvNoResponse } from "./Cluster";
+import { BitFlag, MatterApplicationClusterSpecificationV1_0, TlvBitmap, TlvInt16, TlvUInt16, TlvEnum, TlvField, TlvNullable, TlvObject, TlvSchema, TlvUInt8 } from "@project-chip/matter.js";
+
+/**
+ * Attributes and commands for Temperature Measurement.
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_0} § 2.3
+ */
+export const TemperatureMeasurementCluster = Cluster({
+    id: 0x0402,
+    name: "TemperatureMeasurement",
+    revision: 4,
+    features: { }, // no features
+
+    /** @see {@link MatterApplicationClusterSpecificationV1_0} § 2.3.4 */
+    attributes: {
+       measuredValue: Attribute(0, TlvInt16.bound({ min: -27315, max: 32767 }), { default: 1 }),
```

Comment:
Please see IlluminanceMeasurement ... I adjusted anything which was wrong there ... in fact basically:
* Quality "X" means to add a "tlvNullable around the type
* when having "semantic/dyna,ic ranges defined by other settings then I would not define them, unless they can provide an actual min or max value
* "O" in column Conformance means Optiona so use OptionalAttribute
* Please use default values on if they are really also defined
* Adding documentation per Attribute would also be cool especially relevant I thinkfor the measuredValue because is special

Reply:
Thanks !- Just starting to make sense of the standard

Reply:
Will work a bit today to clean up WaterContent and make sure others are correct